verification/curator-service/ui/cypress/integration/components/SourceTableTest.spec.ts

```diff
@@ -10,13 +10,23 @@ describe('Sources table', function () {
         cy.contains('Example source').should('not.exist');
 
         cy.get('button[title="Add"]').click();
-        cy.get('input[placeholder="Name"]').clear().type('Example source');
-        cy.get('input[placeholder="URL"]').clear().type('www.example.com');
+        cy.get('input[placeholder="Name"]').type('Example source');
+        cy.get('input[placeholder="URL"]').type('www.example.com');
         cy.get('div[data-testid="format-select"]').click();
         cy.contains('li', 'JSON').click();
+        /* This doesn't work and I HAVE NO IDEA WHY PLEASE HELP.
+        cy.get('div[data-testid="op-select"]').scrollIntoView().click();
```

Comment:
Use the `.click({ force: true })`, Luke?

Reply:
Tried that (was even suggested by cypress in the error messages), but it didn't help.